cornflow/models/roles.py

```diff
@@ -173,6 +186,42 @@ def is_service_user(user_id):
 
         return False
 
+    @staticmethod
+    def check_if_role_assigned(user_id, role_id):
+        """
+        Method to check if a user has a given role assigned
+
+        :param int user_id: id of the specific user
+        :param int role_id: id of the specific role
+        :return: a boolean if the user has the role assigned
+        :rtype: bool
+        """
+        user_role = UserRoleModel.get_one_user_role(user_id, role_id)
+        if user_role is None:
+            return False
+        else:
+            return True
+
+    @staticmethod
+    def check_if_role_assigned_disabled(user_id, role_id):
+        """
+        Method to check if a user has a given role assigned but disabled
+
+        :param user_id: id of the specific user
+        :param role_id: id of the specific role
+        :return: a boolean if the user has the role assigned but disabled
+        :rtype: bool
+        """
+        user_role = UserRoleModel.query.filter(
+            UserRoleModel.user_id == user_id,
+            UserRoleModel.role_id == role_id,
+            UserRoleModel.deleted_at != None,
+        ).first()
```

Comment:
I think you can chain queries in sqlalchemy like:
```
user_role = UserRoleModel.get_one_user_role(user_id, role_id).query(UserRoleModel.deleted_at != None)
```
or something similar. Check the examples we have in `meta_model.py` (`get_all_objects`)

Reply:
I will try, but as` UserRoleModel.get_one_user_role` has a` first()` at the end I do not know if we can chain it

Reply:
As I tought, having the `first()` doesnt allow to chain queries.